grammar/BrightScript.g4

```diff
@@ -50,6 +51,15 @@ associativeElementInitializer
     : (identifier | reservedWord | stringLiteral) COLON assignableExpression
     ;
 
+conditionalCompilationStatement
+    : CONDITIONAL_CONST untypedIdentifier EQUALS expression
+    | CONDITIONAL_ERROR .*?
+    | CONDITIONAL_IF expression
+    | CONDITIONAL_ELSEIF expression
+    | CONDITIONAL_ELSE expression
```

Comment:
@vannuysm 

maybe else should not have an expression?

Reply:
Good catch!